cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -490,7 +492,13 @@ instance ( ConvertRawHash blk
           "This block no longer in the VolatileDB and isn't in the ImmutableDB\
           \ either; it wasn't part of the current chain. Block: " <> renderRealPoint pt
         ChainDB.SwitchBackToVolatileDB ->  "SwitchBackToVolatileDB"
-      ChainDB.TraceImmutableDBEvent _ev ->  "TraceImmutableDBEvent"
+      ChainDB.TraceImmutableDBEvent ev -> case ev of
+        ImmDB.ChunkValidationEvent e -> case e of
+          ImmDB.ValidatingChunk c -> "Validating chunk no. " <> showT c
+          ImmDB.ValidatedChunk c c' ->
+            "Validated chunk no. " <> showT c <> " out of " <> showT c'
+            <> ". Progress: " <> pack (showFFloat (Just 2) (100 * fromIntegral (chunkNoToInt c) / fromIntegral (chunkNoToInt c') :: Float) "") <> "%"
```

Comment:
Could we shorten this line?

Reply:
Maybe we could use more descriptive names for `c` and `c'`.

Reply:
What's the role of that `""`?

Reply:
Reworded this into a function. The `""` at the end is becasue `showFFloat` is `StringS`